docs/integrations/spark.md

```diff
@@ -81,6 +184,181 @@ df.write.partitionBy("example-column").parquet(s"s3a://${repo}/${branch}/output-
 
 The data is now created in lakeFS as new changes in your branch. You can now commit these changes, or revert them.
 
+
+## Access lakeFS using the lakeFS-specific Hadoop FileSystem
+
+To use this mode you configure the Spark application to perform metadata operations on the
+lakeFS server, and all data operations directly through the same underlying object store that
+lakeFS uses.  The lakeFS FileSystem currently supports using only the S3A Hadoop FileSystem
+for data access.  In this mode the Spark application will directly read and write from the
+underlying object store, significantly increasing application scalability and performance by
+reducing the load on the lakeFS server.
+
+Accessing data in lakeFS from Spark is the same as accessing S3 data from Spark.  The only
+changes we need to perform are:
+1. Configure Spark to access lakeFS for metadata and S3 or a compatible underlying object
+   store to access data.
+1. Use `lakefs://repo/ref/path/to/data` URIs to read and write data on lakeFS, rather than
+   `s3a://...` URIs.
+
+### Configuration
+
+In order to configure Spark to work using the lakeFS Hadoop FileSystem, you will need to load
+the filesystem JARs and then configure both that FileSystem and the underlying data access
+FileSystem.
+
+#### Load the FileSystem JARs
+
+Add the package `io.lakefs:lakefsfs:<VERSION>` to your Spark job.  For instance, add
```

Comment:
the current code holds `io.lakefs:hadoopfs:0.1.0` we wanted hadoop-lakefs as I recall

Reply:
Done!